client/src/dojo/modelManager/TileManager.ts

```diff
@@ -218,4 +218,17 @@ export class TileManager {
         }, 2000);
       });
   };
+
+  placeStructure = async (entityId: ID, structureType: StructureType, hexCoords: HexPosition) => {
+    if (structureType == StructureType.Hyperstructure) {
+      await this.dojo.systemCalls.create_hyperstructure({
+        signer: this.dojo.network.burnerManager.account!,
+        creator_entity_id: BigInt(entityId || 0),
+        coords: {
+          x: hexCoords.col.toString(),
+          y: hexCoords.row.toString(),
```

Comment:
maybe we can remove all "toString()" here ? since x and y needs to be number (is present in other places in the file)